server/src/main/java/io/pravega/schemaregistry/server/rest/resources/GroupResourceImpl.java

```diff
@@ -0,0 +1,690 @@
+                                             log.warn("Group {} not found", group);
+                                             return Response.status(Status.NOT_FOUND).build();
+                                         }
+                                         log.warn("getGroupProperties for group {} failed with exception: ", group, Exceptions.unwrap(exception));
+                                         return Response.status(Status.INTERNAL_SERVER_ERROR).build();
+                                     }))
+                .thenApply(response -> {
+                    asyncResponse.resume(response);
+                    return response;
+                });
+    }
+    
+    @Override
+    public void getGroupHistory(String group, String namespace, AsyncResponse asyncResponse) {
+        log.info("Get group history called for group {}", group);
+        String resource = Strings.isNullOrEmpty(namespace) ? getGroupResource(group) :
+                getGroupResource(group, namespace);
+        withCompletion("getGroupHistory", READ, resource, asyncResponse,
+                () -> getRegistryService().getGroupHistory(namespace, group, null)
+                                     .thenApply(history -> {
+                                         GroupHistory list = new GroupHistory()
+                                                 .history(history.stream().map(ModelHelper::encode)
+                                                                 .collect(Collectors.toList()));
+                                         log.info("getGroupHistory: {} schemas found for group {}", list.getHistory().size(), group);
+                                         return Response.status(Status.OK).entity(list).build();
+                                     })
+                                     .exceptionally(exception -> {
+                                         if (Exceptions.unwrap(exception) instanceof StoreExceptions.DataNotFoundException) {
+                                             log.warn("Group {} not found", group);
+                                             return Response.status(Status.NOT_FOUND).build();
+                                         }
+
+                                         log.warn("getGroupHistory failed with exception: ", Exceptions.unwrap(exception));
+                                         return Response.status(Status.INTERNAL_SERVER_ERROR).build();
+                                     }))
+                .thenApply(response -> {
+                    asyncResponse.resume(response);
+                    return response;
+                });
+
+    }
+
+    @Override
+    public void updateCompatibility(String group, UpdateCompatibilityRequest updateCompatibilityRequest, 
+                                            String namespace, AsyncResponse asyncResponse) {
+        log.info("Update compatibility called for group {} with new request {}", group, updateCompatibilityRequest);
```

Comment:
trace request here and in other methods.

Reply:
we dont need to "trace" as we are logging it at info level.